seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -311,28 +312,114 @@ def test_validate_anvil_vcf(self, mock_file_iter, mock_file_exist, mock_utils_lo
         self.assertEqual(response.url,
                          '/login/google-oauth2?next=/api/create_project_from_workspace/my-seqr-billing/anvil-no-project-workspace1/validate_vcf')
 
-    @mock.patch('seqr.models.Project._compute_guid', lambda project: f'P_{project.name}')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.id_token.fetch_id_token', lambda *args: MOCK_TOKEN)
-    @mock.patch('seqr.views.utils.airtable_utils.AIRTABLE_API_KEY', MOCK_AIRTABLE_KEY)
-    @mock.patch('seqr.views.utils.airtable_utils.AIRTABLE_URL', MOCK_AIRTABLE_URL)
-    @mock.patch('seqr.views.apis.anvil_workspace_api.AIRFLOW_WEBSERVER_URL', MOCK_AIRFLOW_URL)
-    @mock.patch('seqr.views.apis.anvil_workspace_api.BASE_URL', 'http://testserver/')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.ANVIL_LOADING_DELAY_EMAIL', None)
-    @mock.patch('seqr.views.utils.airtable_utils.logger')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.datetime')
+    @mock.patch('seqr.views.apis.anvil_workspace_api.ANVIL_LOADING_DELAY_EMAIL', 'We are unable to load your data at this time.')
+    @mock.patch('seqr.views.apis.anvil_workspace_api.ANVIL_LOADING_EMAIL_DATE', '2021-06-01')
+    @mock.patch('seqr.views.apis.anvil_workspace_api.mv_file_to_gs', lambda *args, **kwargs: True)
+    @mock.patch('seqr.views.apis.anvil_workspace_api.load_uploaded_file', lambda *args, **kwargs: LOAD_SAMPLE_DATA)
     @mock.patch('seqr.views.apis.anvil_workspace_api.logger')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.load_uploaded_file')
+    @mock.patch('seqr.views.apis.anvil_workspace_api.datetime')
     @mock.patch('seqr.views.apis.anvil_workspace_api.send_html_email')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.safe_post_to_slack')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.mv_file_to_gs')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.tempfile.NamedTemporaryFile')
     @responses.activate
-    def test_create_project_from_workspace(self, mock_tempfile, mock_mv_file, mock_slack,
-                                           mock_send_email, mock_load_file, mock_api_logger, mock_datetime,
-                                           mock_airtable_logger, mock_utils_logger):
+    def test_create_project_from_workspace_loading_delay_email(
```

Comment:
You misunderstood my last PR feedback. The `LoadAnvilDataAPITest` class needs to include this test as well as the new test so they share behavior, and all the shared behavior is scoped to that class level. It is also not at all okay to change the fixtures for this test. It is crucial to test that the behavior for this endpoint has not changed

Reply:
I can move the `test_create_project_from_workspace_loading_delay_email` into the `LoadAnvilDataAPITest` class and add similar test for the `add_workspace_data` API. I don't fully understand what you mean by `to change the fixtures for this test`. Which fixtures are changed in my tests?